### DISCUSSION TITLE
Neo as a Permissioned Blockchain

### DISCUSSION THREAD

**Discussion by krisums** [neo-project/neo#2401] [Q&A] [answered]
Can a private node blockchain of Neo be used as a Permissioned blockchain in production?

**Comment by cschuchardt88** [comment, marked as answer]
you can have a look at https://governance.neo.org/#/ its close to what your looking for.

**Comment by vncoelho** [comment]
@krisums, yes, 

A Permissioned blockchain can be created with the specific governance you want.
Even adding new parameters to the native governance contract or using the current ones.
